django/middleware/csrf.py

```diff
@@ -14,7 +14,8 @@
 from django.utils.cache import patch_vary_headers
 from django.utils.crypto import constant_time_compare, get_random_string
 from django.utils.encoding import force_text
-from django.utils.http import same_origin
```

Comment:
There are some uses on GH: https://github.com/search?q=from+django.utils.http+import+same_origin&ref=reposearch&type=Code&utf8=%E2%9C%93

But it's definitely not documented in the Django docs.

I assume we can remove it and it'd be worth adding a note into the documentation that it was removed and was undocumented?